packages/p2p/src/components/buy-sell/popup.jsx

```diff
@@ -0,0 +1,207 @@
+                                                            setFieldValue('send', send_amount);
+                                                            handleChange(e);
+                                                        }}
+                                                        required
+                                                    />
+                                                )}
+                                            </Field>
+                                        </div>
+                                        <div className='buy-sell__popup-info'>
+                                            <span className='buy-sell__popup-info--title'>{ad.type === 'buy' ? localize('Seller') : localize('Buyer')}</span>
+                                            <p className='buy-sell__popup-info--text'>{ad.advertiser}</p>
+                                        </div>
+                                        <div className='buy-sell__popup-info'>
+                                            <span className='buy-sell__popup-info--title'>{localize('Payment method')}</span>
+                                            <p className='buy-sell__popup-info--text'>{ad.payment_method}</p>
+                                        </div>
+                                        <div className='buy-sell__popup-info'>
+                                            <span className='buy-sell__popup-info--title'>{localize('Advertiser notes')}</span>
+                                            <p className='buy-sell__popup-info--text'>{ad.advertiser_note}</p>
+                                        </div>
+                                    
+                                    </div>
+                                </ThemedScrollbars>
+                                <div className='buy-sell__popup-footer'>
+                                    <Button secondary type='button' onClick={onCancel}>{localize('Cancel')}</Button>
+                                    <Button is_disabled={isSubmitting} primary>{localize('Confirm')}</Button>
+                                </div>
+                            </Form>
+                        )}
+                    </Formik>
+                </div>
+            </Fragment>
+        );
+    }
+
+    validatePopup = (values) => {
+        const { send_validation, receive_validation } = this.getInitialValues();
+
+        const validations = {
+            send: [
+                v => !!v,
+                v => v >= send_validation,
+            ],
+            receive: [
+                v => !!v,
+                v => v >= receive_validation,
+            ],
+        };
+
+        const mappedKey = {
+            send   : localize('Send'),
+            receive: localize('Receive'),
+        };
+
+        // TODO: [translation] text wont pass in the translation script
+
+        const common_messages  = [
+            '{{field_name}} is required',
+            '{{field_name}} below minimum value',
```

Comment:
String extractions don't work like this. The string needs to be inside `localize()` or inside the prop `i18n_default_text` of `<Localize />`.